scripts/static/flatten-romdir.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
This is the main workhorse script. It scans for rom files in a directory and creates hardlinks as a flat list in another. Does its best to avoid creating duplicate entries and being relatively performant.

Reply:
maintainability of this solution is my main concern, maybe this lives separately in its own repo where you maintain it and retronas pulls it in as part of install functionality, or it becomes an optional install